### DISCUSSION TITLE
Support to display no text in table cells

### DISCUSSION THREAD

**Discussion by MA-MacDonald** [grafana/grafana#49046] [Dashboards, Panels & Visualizations]
**What would you like to be added:**
I would like the option to display no text in a table cell

![image](https://user-images.githubusercontent.com/7886459/168718769-8370dbb5-9712-49cb-b033-5d90c9855b21.png)

Ideally I would like to have the following image without the number text showing the the cells.

![image](https://user-images.githubusercontent.com/7886459/168719148-f717378a-7a42-428a-bd37-38bad2de9319.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
